python/test/unit/language/test_core.py

```diff
@@ -773,7 +773,7 @@ def kernel(X, Z, BITCAST: tl.constexpr):
         assert to_numpy(z_tri) == z_ref
 
 
-@pytest.mark.parametrize("dtype_str", [dtype_str for dtype_str in torch_dtypes])
```

Comment:
@Jokeren can you run the test cases again i tried to fix the issue with the function here

Reply:
@Jokeren can you run the testcases again, there was a typo in the function name. i have fixed it now